examples/node-no-lockfile/index.js

```diff
@@ -1 +0,0 @@
-console.log("Hello from Node");
```

Comment:
Unrelated change?

Reply:
Nope.

Reply:
How is this not unrelated?

Reply:
We don't need this file anymore.

Reply:
Alright. I have just seen that you have changed the example from no-lockfile to no-lockfile-canvas. I guess that's fine.